plugins/vestige/vestige.cpp

```diff
@@ -259,7 +259,7 @@ void vestigeInstrument::loadFile( const QString & _file )
 	TextFloat * tf = TextFloat::displayMessage(
 			tr( "Loading plugin" ),
 			tr( "Please wait while loading VST-plugin..." ),
-			PLUGIN_NAME::getIconPixmap( "logo", 24, 24 ), 0 );
+			embed::getIconPixmap( "logo", 24, 24 ), 0 );
```

Comment:
I don't think this and other lines loading `logo.png` should change, as they're about loading the plugin's logo, not a shared resource, right?